frontend/src/scenes/experiments/ExperimentPreview.tsx

```diff
@@ -270,41 +282,57 @@ export function ExperimentPreview({
-                                .map((event: ActionFilterType, idx: number) => (
-                                    <Col key={idx} className="mb-2">
-                                        <Row style={{ marginBottom: 4 }}>
-                                            <div className="preview-conversion-goal-num">
-                                                {experimentInsightType === InsightType.FUNNELS
-                                                    ? (event.order || 0) + 1
-                                                    : idx + 1}
-                                            </div>
-                                            <b>
-                                                <InsightLabel
-                                                    action={event}
-                                                    showCountedByTag={experimentInsightType === InsightType.TRENDS}
-                                                    hideIcon
-                                                    showEventName
-                                                />
-                                            </b>
-                                        </Row>
-                                        {event.properties?.map((prop: AnyPropertyFilter) => (
-                                            <PropertyFilterButton key={prop.key} item={prop} />
-                                        ))}
-                                    </Col>
-                                ))}
+                            <MetricDisplay filters={experiment.filters} />
                             {experiment?.start_date && (
-                                <div className="mb-2 mt-4">
-                                    <LemonButton type="secondary" onClick={openExperimentGoalModal}>
-                                        Change experiment goal
-                                    </LemonButton>
-                                </div>
+                                <>
+                                    <div className="mb-2 mt-4">
+                                        <LemonButton type="secondary" onClick={openExperimentGoalModal}>
+                                            Change experiment goal
+                                        </LemonButton>
+                                    </div>
+                                    {experimentInsightType === InsightType.TRENDS && !experimentCountPerUserMath && (
+                                        <>
+                                            <div className="card-secondary mb-2 mt-4">
+                                                Exposure metric
+                                                <Tooltip
+                                                    title={`This metric determines how we calculate exposure for the experiment. Only users who have this event alongside the property '$feature/${experiment.feature_flag_key}' are included in exposure calculations.`}
```

Comment:
So users can choose to use other events to calculate exposure basically? What's an example case of this from users just curious